### DISCUSSION TITLE
For modded Nintendo Swtch

### DISCUSSION THREAD

**Discussion by gusklein** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#36] [Ideas]
Any way to adapt the json file to use on a modded Nintendo Switch? Thanks!

**Comment by cwmonkey** [comment]
My response for the sake of visibility:

> I have no idea, but if you have access to the file system you can hunt around for the SEASON_DATA_CACHE.JSON, replace it, go offline and try it. You might want to check with Nintendo Switch modding communities to see if they have some idea of how to do that. Holler if you hear anything!

**Reply by gusklein** [reply]
Yeeeei! Have something special to share! It works on Nintendo Switch!! Easiest way to do is use DBI nro app, open game saves, look for No Mans Sky SD.Cache folder and paste the desired expedition JSON file there. Start the game and voilà! I had one small problem with expedition 14, them I'v downloaded again from your expedition generator with default options plus milestones corrected for offline and worked! Thanks!

**Reply by cwmonkey** [reply]
Awesome, thanks for sharing! I'll update my site. @leodium you may want to add this info to the appropriate places in your repo as well.

**Reply by cwmonkey** [reply]
ok, added the info here: https://cwmonkey.github.io/nms-expeditions/

I don't have a Switch nor do I have any insight into the technique, so I just linked to this thread in case anyone has questions 😅 I tried to make it as simple as possible, holler if I need to add anything.

**Reply by ThunderZOD** [reply]
> Yeeeei! Have something special to share! It works on Nintendo Switch!! Easiest way to do is use DBI nro app, open game saves, look for No Mans Sky SD.Cache folder and paste the desired expedition JSON file there. Start the game and voilà! I had one small problem with expedition 14, them I'v downloaded again from your expedition generator with default options plus milestones corrected for offline and worked! Thanks!

Hello, i just can't figure out how to paste the expedition json file on my moded Nintendo switch using DBI or Edizon . Please help.

**Reply by ThunderZOD** [reply]
Hello

> Yeeeei! Have something special to share! It works on Nintendo Switch!! Easiest way to do is use DBI nro app, open game saves, look for No Mans Sky SD.Cache folder and paste the desired expedition JSON file there. Start the game and voilà! I had one small problem with expedition 14, them I'v downloaded again from your expedition generator with default options plus milestones corrected for offline and worked! Thanks!

Hello, I've figured it out, how to paste the expedition .json file onto the cache folder  Open DBI instller on Switch then connect your Switch with your PC using USB cable. Then select "Run MTP responder" from Select menu on DBI from your Switch. Then select "Save" drive from your PC, search for "No Man's Sky" folder, there you can see "Save0000.cache". Go inside that folder and paste the .json file there. After that  when you start NMS you can play the Expedition using your current save file (Go to space Anomaly) or start a new game slot.

**Comment by leodium** [comment]
Je viens de l'ajouter avec le mention Merci à @gusklein =)
_____________________________
.: Boris Verrechia :.

✉ www.leodium.net
✉ rue de Brivioulle, 18 - 4260 Braives
▲ 0473/82.86.56


.: Leodium.est.Fauto/Graphe :.

Photo/Graph(i)er l'Ombre et la Lumière
Projets exposés : *#NoOneInMyWorld **#SomeOneInMyWorld **#Sensu.Elle&Lui*

www.instagram.com/leodium.est.fautographe





Le jeu. 14 nov. 2024 à 09:36, Gerald Burns ***@***.***> a
écrit :
